manifests/base/application-controller/argocd-application-controller-statefulset.yaml

```
@@ -41,6 +41,11 @@ spec:
             port: 8082
           initialDelaySeconds: 5
           periodSeconds: 10
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+              - all
```

```suggestion
              - ALL
```

This value should be in all caps as per spec:

https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted

In addition, the kyverno and aquasecurity (rego) policies check for `ALL` not `all`